Doc/using/android.rst

```diff
@@ -40,8 +40,14 @@ If you're sure you want to do all of this manually, read on. You can use the
 :source:`testbed app <Android/testbed>` as a guide; each step below contains a
 link to the relevant file.
 
-* Build Python by following the instructions in :source:`Android/README.md`.
-  This will create the directory ``cross-build/HOST/prefix``.
+* Either:
```

Comment:
This doesn't describe what the reader hopes to accomplish at this point. Perhaps:
```suggestion
* First, acquire a release build of Python for Android.
  There are two methods of doing so, either downloading a pre-built binary, or compiling Python from the source code yourself.
```

Obviously the sub-bullets would need adjusting